gtest-parallel

```diff
@@ -393,7 +395,7 @@ if options.failed:
 tests.sort(reverse=True, key=lambda x: ((1 if x[0] is None else 0), x))
 
 # Repeat tests (-r flag).
-tests *= options.repeat
+tests = list(itertools.product(tests, range(1, options.repeat + 1)))
```

Comment:
If I'm not mistaken, this is the same as
`  tests = [(t, i + 1) for t in tests for i in range(options.repeat)]
`
Given that itertools.product isn't *that* well-known, it may be better to use the alternative.